README.md

```diff
@@ -1,392 +1,310 @@
-# Learnosity SDK - Java
+<p align="center"><img width="50%" height="50%" src="docs/images/image-logo-graphic.png"></p>
+<h1 align="center">Learnosity SDK - Java</h1>
+<p align="center">Everything you need to start building your app in Learnosity, with the Java programming language.<br> 
+(Prefer another language? <a href="https://help.learnosity.com/hc/en-us/sections/360000194318-Server-side-development-SDKs">Click here</a>)<br>
+An official Learnosity open-source project.</p>
 
-[![Build Status](https://travis-ci.com/Learnosity/learnosity-sdk-java.svg?branch=master)](https://travis-ci.com/Learnosity/learnosity-sdk-java)
+[![Latest Stable Version](https://badge.fury.io/gh/Learnosity%2Flearnosity-sdk-java.svg)](https://github.com/Learnosity/learnosity-sdk-java/releases)
+[![Build Status](https://travis-ci.org/Learnosity/learnosity-sdk-java.svg?branch=master)](https://app.travis-ci.com/github/Learnosity/learnosity-sdk-java)
+[![License](docs/images/apache-license.svg)](LICENSE.md)
+[![Downloads](docs/images/downloads.svg)](https://github.com/Learnosity/learnosity-sdk-java/releases)
+---
 
-Include this library into your own codebase to ease integration with any of the Learnosity APIs.
+## Table of Contents
 
-It supports current [Oracle LTS
-JDKs](http://www.oracle.com/technetwork/java/javase/downloads/eol-135779.html),
-as well as [OpenJDK](http://openjdk.java.net/) with matching versions.
+* [Overview: what does it do?](#overview-what-does-it-do)
+* [Requirements](#requirements)
+* [Installation](#installation)
+* [Quick start guide](#quick-start-guide)
+* [Next steps: additional documentation](#next-steps-additional-documentation)
+* [Contributing to this project](#contributing-to-this-project)
+* [License](#license)
+* [Usage tracking](#usage-tracking)
+* [Further reading](#further-reading)
 
-| Version			| End of Life   |
-|-------------------------------|:-------------:|
-| Oracle JDK8 / OpenJDK 1.8	| 2019-01	|
-| Oracle JDK11			| TBA		|
+## Overview: what does it do?
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 0; Below](https://app.codacy.com/gh/Learnosity/learnosity-sdk-java/pullRequest?prid=8556170)
```suggestion
## Overview: what does it do?

```